src/main/kotlin/com/terraformation/backend/support/api/SupportController.kt

```diff
@@ -53,23 +56,29 @@ class SupportController(private val service: SupportService) {
   }
 
   @ApiResponse(responseCode = "200")
+  @ApiResponse413
+  @ApiResponse415
   @Operation(
       summary = "Upload a temporary attachment.",
       description =
           "Uploads an attachment, which can be assigned to a support request during submission.")
   @PostMapping("/attachment", consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
   @io.swagger.v3.oas.annotations.parameters.RequestBody(
-      content = [Content(encoding = [Encoding(name = "file", contentType = MediaType.ALL_VALUE)])])
+      content =
+          [
+              Content(
+                  encoding =
+                      [Encoding(name = "file", contentType = SUPPORTED_CONTENT_TYPES_STRING)])])
   fun uploadAttachment(
       @RequestPart("file", required = true) file: MultipartFile,
   ): UploadAttachmentResponsePayload {
-    val sizedInputStream =
```

Comment:
I moved away from SizedInputStream because SpringBoot's `MediaType` doesn't really have support for the `video/*` types. So I am checking the header strings directly in this case.

Reply:
I'm curious about what problems you ran into. There aren't predefined constants but
```kotlin
MediaType.parseMediaType("video/mpeg").type
```
returns `"video"` for me.

Reply:
My mistake! I couldn't find the constants so I assumed it would not work accordingly. Updated to use MediaType.type.